x11rb-protocol/Cargo.toml

```diff
@@ -20,15 +20,15 @@ serde = { version = "1", features = ["derive"], optional = true }
 [dev-dependencies]
 criterion = "0.4"
 
-[target.'cfg(unix)'.dependencies.nix]
-version = "0.26"
+[target.'cfg(unix)'.dependencies.rustix]
+version = "0.38"
 default-features = false
-features = ["fs"]
+features = ["std"]
```

Comment:
What exactly does the `std` feature do? I tried removing this and neither `cargo build -p x11rb-protocol` nor `cargo build -p x11rb-protocol --all-features` didn't complain, so either I am missing something or we don't need this.

Edit: Ahh! Thinks started failing in x11rb. So, ignore this comment.
```
error[E0308]: mismatched types
   --> x11rb/src/rust_connection/stream.rs:546:8
    |
546 |     Ok(socket)
    |     -- ^^^^^^ expected struct `std::os::fd::OwnedFd`, found struct `rustix::fd::OwnedFd`
    |     |
    |     arguments to this enum variant are incorrect
    |
    = note: struct `rustix::fd::OwnedFd` and struct `std::os::fd::OwnedFd` have similar names, but are actually distinct types
```